CHANGELOG.md

```diff
@@ -81,7 +81,7 @@ _What's new?_
   - Secrets Manager Checks (`steampipe check benchmark.secretsmanager`) ([#85](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/85))
 - New controls added:
   - ec2_instance_older_generation ([#79](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/79))
-  - lambda_function_with_graviton2 ([#82](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/82))
+  - lambda_function_with_graviton ([#82](https://github.com/turbot/steampipe-mod-aws-thrifty/pull/82))
```

Comment:
@bluedoors Can you please remove this change? The CHANGELOG entry reflects what we added at the time, but we'll call out the rename in the CHANGELOG entry for the next version.

Reply:
@cbruno10 Reverted to the previous CHANGELOG